_pages/en_US/installing-boot9strap-(fredtool).txt

```diff
@@ -42,19 +42,19 @@ If you see a configuration menu, you already have CFW, and continuing with these
 
 #### Section III - Fredtool
 
-1. Open [Fredtool](https://3ds.nhnarwhal.com/3dstools/fredtool.php) on your computer
-1. Select your `movable.sed` file for the "Your movable.sed" field
-1. Select your DSiWare Backup (`<8-character-id>.bin`) file for the "Your dsiware.bin" field
-1. Select "Verify files"
-1. Select "Build and Download"
-1. Wait for the process to complete
-1. When the process is completed, download your modified DSiWare archive from the site
-  + This file contains 2 DSiWare backup files, one clean (unmodified) and one hax (exploited)
+1. Open the [DSIHaxInjector_new](https://jenkins.nelthorya.net/job/DSIHaxInjector_new/build?delay=0sec) website on your computer
+1. Under the "Username" field, enter any alphanumeric name (no spaces or special characters)
+1. Under the "DSiBin" field, upload your `42383841.bin` file using the first "Browse..." option
+1. Under the "MovableSed" field, upload your `movable.sed` file using the second "Browse..." option
+1. Under the "InjectionTarget" field, set the injection target to `DSinternet` (NOT memorypit)
+1. Click "Build"
+1. Wait a few seconds, then click "Last build (#number) x sec ago"
```

Comment:
This could be an issue if someone else built at the same time. might be worth it to go into more detail on finding your own

Reply:
This is also an issue in the TWN guide lol
*sigh* how does one even explain this

Reply:
1. In the Build History section on the left, type the Username into the "Filter Builds" field

Reply:
Something like this OK? 
```
1. Click the "output_(name).zip" link
  + If the `name` does not match the Username you entered, click through each Build history (the `#number` links) until you find your Username
```

Reply:
nvm I saw your reply late lol

Reply:
That look okay? I added some steps to both fredtool pages about the search results